### PR TITLE
flutter app icon and name

### PR DESCRIPTION
**Pull request by inderjeet-developer7**
App icon changed

### CODE REVIEW

File: app/android/app/build.gradle [hunk 1]
```diff
@@ -44,8 +44,8 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.effektio"
```
Comment (gnunicorn) [inline app/android/app/build.gradle:46]:
```suggestion
        applicationId "org.effektio.app"
```

File: app/android/app/build.gradle [hunk 2]
```diff
@@ -44,8 +44,8 @@ android {
...
-        minSdkVersion flutter.minSdkVersion
-        targetSdkVersion flutter.targetSdkVersion
+        minSdkVersion 21
+        targetSdkVersion 29
```
Comment (gnunicorn) [inline app/android/app/build.gradle:48]: what is this about?

**Review by gnunicorn — CHANGES_REQUESTED** [review]
The file app/ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-1024x1024@1x.png should be replaced with a higher resolution one

File: app/android/app/build.gradle (hunk 2 above)
Comment (inderjeet-developer7) [reply app/android/app/build.gradle:48]: For generating the icons for android we need min SDK version as 21 or higher but the version here is lesser so did the changes, Flutter commands give error to these..

**Review by gnunicorn — APPROVED** [review]

**Pull request merged**